tests/Integration/inc/Engine/Admin/Beacon/Beacon/insertScript.php

```diff
@@ -54,22 +55,24 @@ public function testCallbackIsRegistered() {
 		}
 	}
 
-	public function testShouldReturNullWhenNoCapacity() {
-		$this->createUser( 'contributor' );
-		$this->assertFalse( current_user_can( 'rocket_manage_options' ) );
-
-		$this->assertNotContains( 'Beacon', $this->getActualHtml() );
-	}
-
 	/**
 	 * @dataProvider configTestData
 	 */
 	public function testShouldReturnBeaconScript( $config, $expected ) {
 		global $wp_locale;
 
-		$this->createUser( 'administrator' );
-		$this->assertTrue( current_user_can( 'rocket_manage_options' ) );
-
+		if ( true === $config['current_user_can'] ) {
+			$this->createUser( 'administrator' );
+			$this->assertTrue( current_user_can( 'rocket_manage_options' ) );
+		} else {
+			$this->createUser( 'contributor' );
+			$this->assertFalse( current_user_can( 'rocket_manage_options' ) );
+		}
+		$this->white_label = ( $config['white_label'] );
+		if ( null === $expected ) {
+			$this->assertEquals( '', $this->getActualHtml() );
```

Comment:
I think you can use the following assertion
```suggestion
			$this->assertEmpty( $this->getActualHtml() );
```